src/main/java/org/candlepin/sync/Exporter.java

```diff
@@ -176,14 +178,70 @@ public File getFullExport(Consumer consumer, String cdnLabel, String webUrl,
             exportRules(baseDir);
             exportDistributorVersions(baseDir);
             exportContentDeliveryNetworks(baseDir);
-            return makeArchive(consumer, tmpDir, baseDir);
+            return makeArchive(consumer.getUuid(), tmpDir, baseDir);
         }
         catch (IOException e) {
             log.error("Error generating entitlement export", e);
             throw new ExportCreationException("Unable to create export archive", e);
         }
     }
 
+    /**
+     * Creates an entitlement export for a {@link AnonymousCloudConsumer}. If the consumer does not have
+     * an {@link AnonymousContentAccessCertificate} then one will be created.
+     *
+     * @param consumer
+     *  the anonymous cloud consumer to create the export for
+     *
+     * @param serials
+     *  the collection of serials used to filter the {@link AnonymousContentAccessCertificate} by
+     *
+     * @throws ExportCreationException
+     *  if unable to create the compressed entitlement exports
+     *
+     * @return
+     *  the compressed entitlement export
+     */
+    public File createEntitlementExport(AnonymousCloudConsumer consumer, Collection<Long> serials)
```

Comment:
Are we ever going to be exporting an anonymous consumer? Is this something we *want* to be doing? What does the import logic look like for such an entitlement?

Reply:
Probably not! Should we just return a 400 HTTP response for when someone makes a request to GET `/consumers/{consumer_uuid}/certificates` with the header accepting application/zip?

Reply:
We won't be exporting an anon consumer. If they hit this endpoint with `application/zip` mime type, we should be returning a 400